server/tasks/processLostBatchItems.js

```diff
@@ -0,0 +1,51 @@
+const { getLostItems, getBatchItem, deleteItem } = require('../services/items');
+
+module.exports = async (logger) => {
+  try {
+    const lostItems = await getLostItems();
+
+    // Create a map of batchId to lostItems (given many items will be from the same batch, it's more efficient to process them together)
+    const batchIdToLostItems = lostItems.reduce((map, item) => {
+      if (!map[item.batchId]) {
+        map[item.batchId] = [];
+      }
+      map[item.batchId].push(item);
+      return map;
+    }, {});

```

Comment:
there is something up with the formatting of the line endings here - did you maybe copy and paste a segment from a Windows machine? https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types#1552775

Also - I love this pattern of generating a map in a reduce 👍🏼